tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1196,7 +1196,7 @@ func adapterDispatches(promAPI v1.API, adapter string) (float64, error) {
 }
 
 func mixerRequests(promAPI v1.API, svcName, app string) (float64, error) {
-	query := fmt.Sprintf("sum(istio_requests_total{source_app=\"%s\", destination_service_name=\"%s\"})", app, svcName)
+	query := fmt.Sprintf("sum(istio_requests_total{source_app.kubernetes.io/name=\"%s\", destination_service_name=\"%s\"})", app, svcName)
```

Comment:
please revert this part of the changes. The labels used in queries should not be changed.

/cc @gargnupur